### PR TITLE
Update to latest data for all BT tables

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: internal/server/stat/golden/get_stat_set_within_place/CA_County.json [hunk 1]
```diff
@@ -590,21 +590,26 @@
     }
   },
   "metadata": {
-    "2071679530": {
```
Comment (pradh) [inline internal/server/stat/golden/get_stat_set_within_place/CA_County.json:593]: Puzzled why ACS1Year went away

File: internal/server/stat/golden/get_stat_set_within_place_all/CA_County.json [hunk 2]
```diff
@@ -242,913 +242,953 @@
...
           },
-          "meta_hash": 2071679530
+          "meta_hash": 2939218655
         },
         {
           "stat": {
             "geoId/06001": {
-              "date": "2019",
-              "value": 1656754
+              "date": "2011",
```
Comment (pradh) [inline internal/server/stat/golden/get_stat_set_within_place_all/CA_County.json:482]: Puzzled why the date is older now.

Comment (pradh) [reply internal/server/stat/golden/get_stat_set_within_place_all/CA_County.json:482]: Ohh this is bad json diffs I think :(.   Perhaps we should sort or something.

File: internal/server/node/golden/get_property_values/contained_in_place.json [hunk 3]
```diff
@@ -146,7 +146,7 @@
             "City"
           ],
           "dcid": "geoId/0627080",
-          "provenanceId": "dc/sm3m2w3"
+          "provenanceId": "dc/5n63hr1"
```
Comment (pradh) [inline internal/server/node/golden/get_property_values/contained_in_place.json:149]: Need to debug why this is...

File: internal/server/stat/golden/get_stat_set_within_place_all/CA_County.json (hunk 2 above)
Comment (shifucun) [reply internal/server/stat/golden/get_stat_set_within_place_all/CA_County.json]: Sorted by sources now

File: internal/server/node/golden/get_property_values/contained_in_place.json (hunk 3 above)
Comment (shifucun) [reply internal/server/node/golden/get_property_values/contained_in_place.json:149]: @Spaceenter need to debug this in Prophet.

**Comment by shifucun** [comment]
Thanks for review!

**Pull request merged**